core/trino-main/src/main/java/io/trino/operator/aggregation/TypedSet.java

```diff
@@ -76,6 +76,24 @@ public static TypedSet createEqualityTypedSet(
                 elementHashCodeOperator,
                 elementType.createBlockBuilder(null, expectedSize),
                 expectedSize,
+                functionName);
+    }
+
+    public static TypedSet createEqualityTypedSet(
```

Comment:
> Add missing TypedSet factory


what made you think this particular one is "missing"?
add rationale to the commit message.

also, this adds an unused method. it can be added later, when it's actually needed, so i'd drop the commit from here

Reply:
I do use it a few commits later; I can move it there.